Pflichtenheft.md

```diff
@@ -0,0 +1,286 @@
+# Pflichtenheft
+Belegarbeit Software-Engineering II
+
+Sommersemester 2018
+
+Thema: Entwicklung eines SW‐Systems zur Unterstützung der Planung von Präsentationen im Kontext des Moduls SE II
+
+Bearbeitet von:
+* Denis Keiling
+* Fabian Krehnke
+* Leo Lindhorst
+* Đức Hùng Nguyễn
+* Oliver von Seydlitz
+
+Auftraggeber: Prof. Dr. Hauptmann
+
+## Problemstellung und Systemziel
+
+### Problem
+Die Terminbelegung für Prüfungspräsentationen über E-Mail und manuelles Editieren der Excel-Tabelle ist zeitaufwändig und kann zu Fehlern führen.
+
+### Ziel
+Es soll ein Software-System entwickelt werden, über das die Terminplanung geregelt wird. Es soll folgendes gewährleisten:
+* Ein gleichzeitiges Buchen darf nicht möglich sein um Konflikte zu vermeiden
+* Die Termine müssen vom Prüfer flexibel angegeben werden können
+* Organisation muss dezentral sein um den Prüfer zu entlasten
+
+## Systemkontext
+Der Kontext des Systems wird durch folgendes Diagramm beschrieben:
+
+![Kontexdiagramm](diagrams/awf-kontext.jpg)
+
+## Benutzerschnittstelle
+TODO
+
+## Anforderungen
+
+### Funktionale Anforderungen
+
+#### Überblick
+Die folgende Tabelle gibt einen Überblick über die essentiellen Gruppen und essentiellen Funktionen, sowie deren Auslöser und Reaktionen:
+
+Anwendungsfall | Auslöser | Reaktion
+---------------|----------|---------
```

Comment:
Ich denke die Auslöser müssen mehr aus Nutzersicht formuliert sein. Im Sinne von "Startdatum + Anzahl Gruppen **eingeben**" oder "Termine **ausgeben** bzw. Warnhinweis **anzeigen**" sonst wird nicht klar, was das System tut. "Termine" sind ja erstmal keine Reaktion. Denke ich zumindest...

Reply:
Ich glaube auch , dass Termin(neu ) nicht mit in der ART erwähnt werden muss.